acra/configuring-maintaining/general-configuration/acra_log_verifier.md

```diff
@@ -0,0 +1,293 @@
+---
+title: AcraLogVerifier
+bookCollapseSection: true
+weight: 3
+---
+
+# AcraLogVerifier
+
+AcraLogVerifier is command-line utility that verifies secure logs dumped from AcraServer/AcraTranslator/AcraConnector started with `--audit_log_enable=true` flag.
+
+It expects symmetric key to decrypt keys from keystore from  `ACRA_MASTER_KEY` environment variable in pair `--keys_dir` flag or from HashiCorp Vault together with `--vault_*` flags. 
+
+## Command line flags
+
+### Verification
+
+* `--audit_log_file=<filename>`
+
+  Path to log file that should be verified.
+
+* `--audit_log_file_format={plaintext|json|CEF}`
+
+  Expected format of secure log file(s) that should be verified: plaintext, JSON or CEF. 
+  Default is  "plaintext".
+
+* `--audit_log_file_list`
+
+  Path to file with list of files that should be verified. File contain one path per line: 
+  ```
+  <filepath1>
+  <filepath2>
+  ...
+  <filepathN>
+  ```
+
+* `--audit_log_missing_ok`
+
+  Don't fail validation if some file from `audit_log_file_list` cannot be opened. Verifier will log file path with `WARNING` severity level
+  and will not interrupt validation with non-zero exit status.
+
+* `-d`
+
+  Debug mode, shows order of input files. Useful in pair with `--audit_log_file_list` flag. 
+  Verifier will print absolute file paths that in the order used during validation. 
+
+### Logging
+
+* `--logging_format={plaintext|json|CEF}`
```

Comment:
Do we really need to duplicate such information, I remember adding some logging info here:
https://github.com/cossacklabs/product-docs/blob/master/acra/configuring-maintaining/logging/_index.md

Reply:
We have already existing structure of CLI description for our tools with common sections, logging also too. And actually, I missed already existing section in `logging` ) but I think it's okay to duplicate. On separate page we can describe it more verbosely. On CLI pages it more useful when all arguments listed together with brief description. Plus `acra_log_verifier` may use these flags in another way. Especially here this flag specify type of log file that will be used, not logs that will be generated.

Reply:
Agree, makes sense